player/command.c

```diff
@@ -5085,6 +5085,11 @@ static void cmd_stop(void *p)
 
     if (!(flags & 1))
         playlist_clear(mpctx->playlist);
+
+    if (mpctx->opts->player_idle_mode < 2 && mpctx->opts->position_save_on_quit) {
```

Comment:
Is this below 80 columns?
Does any other code use numeric comparison for the idle option?
Otherwise probably good.

Reply:
> Is this below 80 columns?

Fixed

> Does any other code use numeric comparison for the idle option?

E.g. https://github.com/mpv-player/mpv/blob/master/player/loadfile.c#L1835